ibis/backends/pyspark/tests/test_client.py

```diff
@@ -20,3 +23,19 @@ def test_catalog_db_args(con, monkeypatch):
     con.drop_table("t2", database="spark_catalog.default")
 
     assert "t2" not in con.list_tables(database="default")
+
+
+def test_create_table_no_catalog(con, monkeypatch):
+    monkeypatch.setattr(ibis.options, "default_backend", con)
+    t = ibis.memtable({"epoch": [1712848119, 1712848121, 1712848155]})
+
+    # create a table in specified catalog and db
+    con.create_table("t2", database=("default"), obj=t, overwrite=True)
+
+    assert "t2" not in con.list_tables()
+    assert "t2" in con.list_tables(database="default")
+    assert "t2" in con.list_tables(database=("default"))
```

Comment:
Did you mean to write something else here? `"default"` and `("default")` are equivalent Python expressions.

Reply:
oh, that was just unthinking editing -- I'll rip that out